docs/tutorial.md

```diff
@@ -285,6 +285,24 @@ create_keras_neuropod(
     `create_keras_neuropod` runs inference with the test data immediately after creating the neuropod. Raises a `ValueError` if the model output does not match the expected output.
 
 
+### Ludwig
+
+Ludwig contains a utility to export model to the Neuropod format.
+If we train a Ludwig model like this:
+
+```
+ludwig train --data_csv <MY_DATA>PATH> --model_definition <MY_MODEL_DEFINITION> --output_directory <LUDWIG_OUTPUT_DIRECTORY>
+```
+
+We can package it as a Neuropod by running:
+
+```
+python -m ludwig.utils.neuropod_utils --ludwig_model_path <LUDWIG_OUTPUT_DIRECTORY>/<MODEL_NAME>/model --neuropod_path <NEUROPOD_OUTPUT_PATH>
```

Comment:
I see you use "python", is it important for Ludwig model what python version is used, should it be consistent with Neuropod's python (there are different packages)? I personally think we should make it clear to avoid confusions of new users.

Reply:
Ludwig works for python 3.5+, but is tested only on 3.6+, although when working on the deployment on some models in Michelangelo we discovered a but in the embedded python interpreter for 3.6 that was solved in 3.7, so really i should specify to use python 3.7 (haven't tested 3.8 yet). Does that sound good?